controls/datagrid/aggregates/styling.md

```diff
@@ -34,6 +34,87 @@ Use the following properties to style the `GroupHeader`:
 | `TextFontSize` | Defines the size for the text part of the `GroupHeader`. |
 | `TextMargin` | Defines the margin for the text part of the `GroupHeader`. |
 
+### Style the Aggregates in the Group Header
+
+You can style the appearance of the aggregate results based on the value of the `GroupAggregatesAlignment` (`enum` of type `Telerik.Maui.Controls.DataGrid.DataGridGroupAggregatesAlignment`) property.
+
+To style the aggregates in the group header when the value for the `GroupAggregatesAlignment` is `None`, use the `GroupHeaderStyle` (of type `Style` with target type `DataGridGroupHeaderAppearance`) property. 
+
+The available properties in the `GroupHeaderStyle` are:
+
+| Property  | Description |
+| ------ | ------ |
+| `AggregatesTextColor` | Defines the color for the aggregates part of the `GroupHeader` |
```

Comment:
I recommend that you add punctuation consistently at the end of each Description. (Note how the next table has periods at the end of each Description.)